spec/lib/tasks/db_data_migrate_rake_spec.rb

```diff
@@ -125,6 +125,48 @@
       }
     end
 
+    context 'with any storage providers' do
```

Comment:
Should this read 'without any storage providers'?

Reply:
yes